### PR TITLE
Schema compare publish

### PR DESCRIPTION
**Pull request by kisantia**
This adds the publish operation to the toolbar in the schema compare extension.

schemaComparePublish

### CODE REVIEW

**Comment by pensivebrian** [comment]
Do we detect and handle schema drift?

**Comment by udeeshagautam** [comment]
@pensivebrian Do you mean if we support the blocking at drift or how we deploy with drift allowed? 

For the first one - The allowing deployment with drift will come from options (other PR). Currently its just using the default which is BlockWhenDriftDetected = true.

For the second one - I am assuming dacfx will handle it. Do you think additional handling is required?

File: extensions/schema-compare/src/schemaCompareResult.ts [hunk 1]
```diff
@@ -310,11 +315,33 @@ export class SchemaCompareResult {
 		});
 	}
 
+	private createApplyButton(view: azdata.ModelView) {
+
+		this.applyButton = view.modelBuilder.button().withProperties({
+			label: localize('schemaCompare.updateButton', 'Apply'),
```
Comment (udeeshagautam) [inline extensions/schema-compare/src/schemaCompareResult.ts]: Update or Apply?

**Comment by pensivebrian** [comment]
@udeeshagautam - we can handle drift in another PR- I just wanted to make sure it's on our radar.

Today in SSDT, if drift is detected, the script and publish button will disable, and a message will appear to re-run the compare operation. Hopefully this logic is bound to the BlockWhenDriftDetected deploy option.

I'm not sure if drift detection runs occasionally in the background, or just after the compare operation.

File: extensions/schema-compare/src/schemaCompareResult.ts (hunk 1 above)
Comment (kisantia) [reply extensions/schema-compare/src/schemaCompareResult.ts]: Andrew suggested "Apply," but it can always be changed if it's confusing

**Pull request merged**